src/VisualStudio/Core/Def/Implementation/FindReferences/Contexts/AbstractTableDataSourceFindUsagesContext.cs

```diff
@@ -414,7 +419,14 @@ protected RoslynDefinitionBucket GetOrCreateDefinitionBucket(DefinitionItem defi
             }
 
             public sealed override ValueTask ReportMessageAsync(string message, CancellationToken cancellationToken)
-                => throw new InvalidOperationException("This should never be called in the streaming case.");
```

Comment:
this can now be called in the streaming case.  in streaming-go-to-impl, we may end up finding no results.  so we want to show that message to the user in the FAR window.  We could just use the default string, but since hte API allows the caller to specify the message to show hte user, we should respect it.